src/chains.jl

```diff
@@ -612,7 +626,26 @@ function pool_chain(c::Chains)
     return Chains(pool_data, names(c), c.name_map; info=c.info)
 end
 
-# replace parameter names
+"""
+    replacenames(chains::Chains, dict::AbstractDict) 
+
+Replace parameter names by creating a new `Chains` object that shares the same underlying data.
+
+# Example
+```
```

Comment:
This should be written as a proper example, and preferably as a doctest (which should be tested, of course).